core/pom.xml

```diff
@@ -215,6 +215,11 @@
       <version>3.2.10</version>
     </dependency>
     <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-scala_2.10</artifactId>
+      <version>2.3.1</version>
```

Comment:
v2.4.4 is already included in master so if you rebase you won't have to include it again